allennlp/predictors/sentence_tagger.py

```diff
@@ -32,3 +34,59 @@ def _json_to_instance(self, json_dict: JsonDict) -> Instance:
         sentence = json_dict["sentence"]
         tokens = self._tokenizer.split_words(sentence)
         return self._dataset_reader.text_to_instance(tokens)
+
+    @overrides
+    def predictions_to_labeled_instances(self,
+                                         instance: Instance,
+                                         outputs: Dict[str, np.ndarray]) -> List[Instance]:
+        """
+        Imagine an NER model predicts three named entities (each one with potentially
+        multiple tokens). For each individual entity, we create a new Instance that has
+        the label set to only that entity and the rest of the tokens are labeled as outside.
+        We then return a list of those Instances.
+
+        For example:
+        Mary  went to Seattle to visit Microsoft Research
+        U-Per  O    O   U-Loc  O   O     B-Org     L-Org
+
+        We create three instances.
+        Mary  went to Seattle to visit Microsoft Research
+        U-Per  O    O    O     O   O       O         O
+
+        Mary  went to Seattle to visit Microsoft Research
+        O      O    O   U-LOC  O   O       O         O
+
+        Mary  went to Seattle to visit Microsoft Research
+        O      O    O    O     O   O     B-Org     L-Org
+        """
+        predicted_tags = outputs['tags']
+        predicted_spans = []
+
+        i = 0
+        while i < len(predicted_tags):
+            tag = predicted_tags[i]
+            # if its a U, add it to the list
+            if tag[0] == 'U':
+                cur_tags = [t if idx == i else 'O' for idx, t in enumerate(predicted_tags)]
```

Comment:
`current` instead of `cur`.  The additional readability is worth the four extra characters.